### PR TITLE
Changelog for the 1.3.0 release

### PR DESCRIPTION
**Pull request by jrfnl**
* Preliminary release date set at Thursday June 6.
* Includes all currently merged changes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+* PHPCS: New `Yoast.Commenting.CoversTag` sniff.
+    This sniff verifies that:
+    - the contents of a `@covers` annotation is valid based on what's supported by PHPUnit;
+    - there are no duplicate `@covers` or `@coversNothing` tags in a docblock;
+    - a docblock doesn't contain both a `@covers` tag as well as a `@coversNothing` tag;
+    Includes a fixer for common errors.
+* PHPCS: New `Yoast.Commenting.TestsHaveCoversTag` sniff.
+    This sniff verifies that all unit test functions have at least one `@covers` tag - or a `@coversNothing` tag - in the function docblock or in the class docblock.
+* PHPCS: New `Yoast.Yoast.AlternativeFunctions` sniff.
+    This sniff allows for discouraging/forbidding the use of PHP/WP native functions in favour of using Yoast native functions.
```
Comment (igorschoester) [inline CHANGELOG.md]: We should use American spelling.

```suggestion
    This sniff allows for discouraging/forbidding the use of PHP/WP native functions in favor of using Yoast native functions.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+    In this initial version, the sniff checks for the use of the `json_encode()` and `wp_json_encode()` functions and suggests using `WPSEO_Utils::format_json_encode()` instead.
+    Note: this sniff contains an auto-fixer. If for any of the repos, the auto-fixer should not be used, the auto-fixer can be disabled from within the repo specific ruleset using `<rule ref="..." phpcs-only="true"/>`.
+* PHPCS: The `Squiz.WhiteSpace.MemberVarSpacing` sniff.
+    This sniff verifies and auto-fixes the amount of blank lines between property declarations within OO-structures.
```
Comment (igorschoester) [inline CHANGELOG.md]: Blank lines are countable.

```suggestion
    This sniff verifies and auto-fixes the number of blank lines between property declarations within OO-structures.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+#### Changed
+* PHPCS: Files in the following directories will now be excluded from all scans by default:
+    - `/.git/`
+    - `/.wordpress-svn/`
+    - `/node-modules/`
+    - `/vendor/`
+    - `/vendor_prefixed/`
+    Custom repo specific rulesets which contain excludes to this effect, can now remove them safely.
+* PHPCS: The message type for issues reported by the `Generic.Formatting.MultipleStatementAlignment` and `WordPress.Arrays.MultipleStatementAlignment` sniffs, has been upgraded from `warning` to `error`.
+* PHPCS: The WPCS native check for the `json_encode()` function in the `WordPress.WP.AlternativeFunctions` has been disabled in favour of the new YoastCS native `Yoast.Yoast.AlternativeFunctions` sniff.
```
Comment (igorschoester) [inline CHANGELOG.md]: We should use American spelling.

```suggestion
* PHPCS: The WPCS native check for the `json_encode()` function in the `WordPress.WP.AlternativeFunctions` has been disabled in favor of the new YoastCS native `Yoast.Yoast.AlternativeFunctions` sniff.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+* Travis: As there is now a sniff which extends a WPCS sniff, the unit tests will now run against various combinations of PHPCS and WPCS combined.
+* Minor housekeeping.
+
+#### Removed
+* PHPMD is no longer part of the YoastCS repo.
+    PHPMD was not used as a stand-alone tool by any of the repos, only in combination with CodeClimate.
+* Travis: Testing of the repo against PHP `nightly` (PHP 8, unstable) as no viable PHPUnit version is currently available.
+
+#### Fixed
+* PHPCS: The `Yoast.Files.FileName` sniff will now always suggest to remove the longest prefix of the prefixes passed in the configuration.
```
Comment (igorschoester) [inline CHANGELOG.md]: Apparently "suggest to" is only for sentences in question form.

```suggestion
* PHPCS: The `Yoast.Files.FileName` sniff will now always suggest removing the longest prefix of the prefixes passed in the configuration.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
 
 This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).
```
Comment (igorschoester) [inline CHANGELOG.md]: Non-PR related, perhaps still a good place to fix this.

```suggestion
This project adheres to [Semantic Versioning](https://semver.org/) and [Keep a CHANGELOG](https://keepachangelog.com/).
```

Same for line 1 above.
```
# Change Log
```
to
```
# Changelog
```

**Review by igorschoester — CHANGES_REQUESTED** [review]
CR - a couple of language things (thanks Grammarly 😉 ).
All the entries themselves 👍

**Comment by jrfnl** [comment]
@igorschoester Thanks for your careful scrutiny! All fixed up now.

**Comment by igorschoester** [comment]
CR 👍

**Pull request merged**